internal/runners/config/set.go

```diff
@@ -28,7 +27,10 @@ func NewSet(prime primeable) *Set {
 func (s *Set) Run(params SetParams) error {
 	// Cast to rule type if applicable
 	var value interface{}
-	rule := configMediator.GetRule(params.Key.String())
+	rule, found := configMediator.GetRule(params.Key.String())
+	if !found {
+		return locale.NewInputError("unknown_config_key", "Unknown config key: {{.V0}}", params.Key.String())
+	}
```

Comment:
I think what we want here is a `configMediator.KnownRule` or something like that. This is overloading the use-case for `GetRule()`.